test/numbers.jl

```diff
@@ -460,6 +460,8 @@ end
     @test sign(-0//1) == 0
     @test sign(1//0) == 1
     @test sign(-1//0) == -1
+    @test sign(pi) === 1.0
```

Comment:
```suggestion
    @test sign(pi) == 1
```
This allowes different return type in the future. Like for `@test sign(1.0) == 1`